### PR TITLE
added self signed ssl ability

### PR DESCRIPTION
**Pull request by srinivasreddych**
### Description:

This change deploys self signed cert for SSL ability for landing page

### Issue Reference URL

https://github.com/awslabs/aws-eks-data-maker/issues/<NUMBER>

----------
### Do not change content below. Mark applicable check boxes only.

Thank you for submitting a contribution to AWS Orbit Workbench.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a ISSUE associated with this PR?
- [ ] Does your PR title start with ISSUE-XXXX where XXXX is the issue number you are trying to resolve? Pay particular attention to the hyphen "-" character.
- [ ] Has your PR been rebased against the latest commit within the target branch (typically 'main')?
- [ ] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you ensured that the full suite of tests is executed?
- [ ] Have you written or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under Apache License 2.0?

### CODE REVIEW

**Review by abaror — CHANGES_REQUESTED** [review]
you have to update the other manifest or the CICD will fail

File: cli/aws_orbit/remote_files/cert.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+import logging
+from aws_orbit import sh
+import boto3
+from botocore.exceptions import ClientError
+import shlex
+import subprocess
+
+_logger: logging.Logger = logging.getLogger(__name__)
+
+def run_command(cmd):
```
Comment (chamcca) [inline cli/aws_orbit/remote_files/cert.py:11]: missing return. this won't pass type checking

File: cli/aws_orbit/remote_files/cdk/foundation.py [hunk 2]
```diff
@@ -93,6 +96,23 @@ def __init__(self, scope: Construct, id: str, context: "FoundationContext", **kw
 
         self.user_pool: cognito.UserPool = self._create_user_pool()
 
+        # self.artifacts_domain = codeartifact.CfnDomain(
```
Comment (chamcca) [inline cli/aws_orbit/remote_files/cdk/foundation.py:99]: we should eliminate commented code

File: cli/aws_orbit/remote_files/cdk/foundation.py [hunk 3]
```diff
@@ -36,9 +38,10 @@
 
 
 class FoundationStack(Stack):
-    def __init__(self, scope: Construct, id: str, context: "FoundationContext", **kwargs: Any) -> None:
+    def __init__(self, scope: Construct, id: str, context: "FoundationContext", ssl_cert_arn:str, **kwargs: Any) -> None:
```
Comment (chamcca) [inline cli/aws_orbit/remote_files/cdk/foundation.py:41]: ssl_cert:str will likely fail flake8 checking. "ssl_cert:str" -> "ssl_cert: str"

File: cli/aws_orbit/remote_files/cdk/foundation.py [hunk 4]
```diff
@@ -18,14 +18,16 @@
...
 from typing import Any, Dict, List, cast
-
-import aws_cdk.aws_cognito as cognito
-import aws_cdk.aws_kms as kms
-import aws_cdk.aws_s3 as s3
-import aws_cdk.core as core
-from aws_cdk import aws_ec2 as ec2
-from aws_cdk import aws_iam as iam
-from aws_cdk import aws_ssm as ssm
+from aws_cdk import (
```
Comment (chamcca) [inline cli/aws_orbit/remote_files/cdk/foundation.py:21]: i think this import convention will fail our isort and/or flake8 checking

File: cli/aws_orbit/remote_files/cdk/foundation.py (hunk 2 above)
Comment (srinivasreddych) [reply cli/aws_orbit/remote_files/cdk/foundation.py:99]: I thought when we bump the cdk versions, we would just uncomment the codeartifact construct here. Should i remove them totally?

File: samples/manifests/plugins/dev-env-with-plugins.yaml [hunk 5]
```diff
@@ -33,6 +33,7 @@ Networking:
         NodesSubnets: !SSM ${/orbit-foundation/dev-env/resources::NodesSubnets}
     Frontend:
         LoadBalancersSubnets: !SSM ${/orbit-foundation/dev-env/resources::LoadBalancersSubnets}
+        SslCertArn: !SSM ${/orbit-foundation/dev-env/resources::SslCertArn}
```
[the remark this answers is not in the document]
Comment (srinivasreddych) [reply samples/manifests/plugins/dev-env-with-plugins.yaml:36]: Done

File: cli/aws_orbit/remote_files/cert.py (hunk 1 above)
Comment (srinivasreddych) [reply cli/aws_orbit/remote_files/cert.py]: Fixed

File: cli/aws_orbit/remote_files/cdk/foundation.py (hunk 3 above)
Comment (srinivasreddych) [reply cli/aws_orbit/remote_files/cdk/foundation.py]: Done

**Review by abaror — APPROVED** [review]

**Pull request merged**